SQLLexer.g4

```diff
@@ -126,11 +123,28 @@ IDENTIFIER:
     | [A-Z_] [A-Z_0-9]* // Ordinary identifiers
 ; // TODO check: needs more chars in set
 
-NUMERIC_LITERAL: ((DIGIT+ ('.' DIGIT*)?) | ('.' DIGIT+)) ('E' [-+]? DIGIT+)? | '0x' HEX_DIGIT+;
+// literals
 
-BIND_PARAMETER: [@$] IDENTIFIER;
+BOOLEAN_LITERAL:
+    'true'
+    | 'false'
+;
+
+NUMERIC_LITERAL:
+    [0-9]+
+;
+
+BLOB_LITERAL:
+    '0x' [0-9a-f]+
+;
 
-STRING_LITERAL: '\'' ( ~'\'' | '\'\'')* '\'';
+TEXT_LITERAL:
```

Comment:
Why this is not the same as the old `STRING_LITERAL`?

Reply:
Oh I changed this while debugging. Will change back